doc/nrf/known_issues.rst

```diff
@@ -2228,6 +2228,11 @@ nrfxlib
 Crypto
 ======
 
+.. rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+
+NCSDK-20287: CryptoCell backend only support AES GCM with a nonce size of 12 bytes.
```

Comment:
The known issue is that the nonce size != 12 bytes will produce incorrect tag I think not that cryptocell only supports that. So I would suggest to change the text to state this in the title. In the explanation you can write the APIs affected (PSA and legacy mbedtls if they are)